common/src/main/java/whocraft/tardis_refined/client/screen/selections/ShellSelectionScreen.java

```diff
@@ -68,6 +78,23 @@ public static void selectShell(ShellTheme theme) {
     public void render(PoseStack poseStack, int i, int j, float f) {
         renderBackground(poseStack);
 
+
+        ClientLevel lvl = Minecraft.getInstance().level;
+        RandomSource rand = lvl.random;
+
+        boolean isCrashed = TardisClientData.getInstance(lvl.dimension()).isCrashing();
```

Comment:
Would be good to have some general comments for the magic numbers in case we forget later on.